projects/kit/directives/lazy-loading/lazy-loading.directive.ts

```diff
@@ -1,14 +1,29 @@
-import {Directive, ElementRef, HostBinding, Inject, Input} from '@angular/core';
+import {
+    ChangeDetectorRef,
+    Directive,
+    ElementRef,
+    HostBinding,
+    Inject,
+    Input,
+} from '@angular/core';
 import {IntersectionObserverService} from '@ng-web-apis/intersection-observer';
 import {TuiDestroyService} from '@taiga-ui/cdk';
 import {TuiLazyLoadingService} from './lazy-loading.service';
 
```

Comment:
Let's remove this empty line so all imports are connected ;)

Reply:
Finding "Import sources within a group must be alphabetized." combination was actually harder than expected 😅 Done